playbooks/roles/ocp-customization/tasks/main.yaml

```diff
@@ -38,6 +38,10 @@
       when: powervm_rmc
       import_tasks: powervm_rmc.yaml
 
+    - name: Allow SMT configuration via node label (SMT=X)
+      when: ansible_architecture == "ppc64le"
+      import_tasks: smt_configuration.yaml
```

Comment:
Let's make this task optional. I'm not very comfortable fiddling with SMT settings and would like to keep this optional and run through some iterations of e2e tests to verify stability.

Reply:
you mean adding a variable `smt_control` (true|false) in the playbook ?

Reply:
Yes and default to false

Reply:
done in my last commit